queries/comment/highlights.scm

```diff
@@ -10,15 +10,26 @@
   (#any-of? @comment.todo "TODO" "WIP"))
 
 ((tag
-  (name) @comment.note @nospell
+  (name) @comment.info @nospell
   ("(" @punctuation.bracket
     (user) @constant
     ")" @punctuation.bracket)?
   ":" @punctuation.delimiter)
-  (#any-of? @comment.note "NOTE" "XXX" "INFO" "DOCS" "PERF" "TEST"))
+  (#any-of? @comment.info "XXX" "INFO" "DOCS" "PERF" "TEST"))
 
-("text" @comment.note @nospell
-  (#any-of? @comment.note "NOTE" "XXX" "INFO" "DOCS" "PERF" "TEST"))
+("text" @comment.info @nospell
+  (#any-of? @comment.info "XXX" "INFO" "DOCS" "PERF" "TEST"))
+
+((tag
+  (name) @comment.hint @nospell
+  ("(" @punctuation.bracket
+    (user) @constant
+    ")" @punctuation.bracket)?
```

Comment:
This pattern repeats multiple times, I don't think it's necessary to do that :thinking: 

Maybe a follow-up to
```diff
-  ("(" @punctuation.bracket
-    (user) @constant
-    ")" @punctuation.bracket)? ; remove this from any place that has it
+ (tag
+   "(" @punctuation.bracket
+   .
+   (user) @constant
+   .
+   ")" @punctuation.bracket)
```

some day will be nice

Reply:
These patterns are also new and look unrelated to the PR description.

Reply:
Yeah it's unrelated. I'll note this down and do it sometime later.

Reply:
This was intentional, otherwise those symbols would be highlighted in all words that look like a tag (`FOO:`), see https://github.com/nvim-treesitter/nvim-treesitter/pull/4298.